### ISSUE TITLE
Setting paging: false loads only one item

### ISSUE THREAD

**Issue by elie222** [aldeed/meteor-tabular#76]
```
TabularTables.Books = new Tabular.Table({
  name: "BookList",
  collection: Books,
  paging: false,
  ...
})
```
This code will only load one collection item.

**Comment by ajlin500** [comment]
I ran into the same problem

**Comment by cedricmay** [comment]
+1

**Comment by bySabi** [comment]
Same here

**Comment by premosystems** [comment]
same here

**Comment by bipin244** [comment]
same here also

**Comment by CvanRijen** [comment]
+1

**Comment by nitinjadhav466** [comment]
Same Issue here. I am trying to debug the code to see if there is workaround.

**Comment by bySabi** [comment]
@nitinjadhav466 see my PR there is a posible fix ... https://github.com/aldeed/meteor-tabular/issues/107

**Comment by dkoo761** [comment]
+1

**Comment by dkoo761** [comment]
I just discovered a simple solution. In your table definition, set `dom: 'frt'` and `pageLength: 10000`. That will hide the pagination elements in the DOM and give you a big enough page that you'll never need to paginate (if you have more than 10,000 elements in your table without pagination, well... ;).

More details about the `dom` option: https://datatables.net/reference/option/dom

**Comment by bySabi** [comment]
Good!!!
@dkoo761, please create a PR with `dom` tip on README file.

**Comment by Benco1** [comment]
Ugh.

**Comment by aldeed** [comment]
Fixed in 1.3.0
